include/EASTL/chrono.h

```diff
@@ -43,10 +43,6 @@
 	#endif
 	EA_RESTORE_ALL_VC_WARNINGS()
 	#pragma warning(pop)
-#endif
```

Comment:
The intent of this PR appears to be correct (avoid including `<thr/xtimec.h>` for MSVC), but this preprocessor coalescing appears to be incorrect. Note how left line 48 indicates that MICROSOFT and MINGW can be defined simultaneously. Previously, MICROSOFT and MINGW would activate line 29, and separately would skip left line 48 and activate left line 55. Now, the coalescing means that line 29 will absorb the MICROSOFT and MINGW scenario.

I believe that this PR should be changed to be non-intrusive, by simply replacing the `<thr/xtimec.h>` include with a do-nothing comment, and not coalescing the preprocessor logic.

Reply:
I agree; presumably CI didn't catch this due to the absence of MinGW coverage.